pkg/api/version/version.go

```diff
@@ -0,0 +1,30 @@
+package version
+
+import (
+	version "github.com/mcuadros/go-version"
+	"github.com/rancher/norman/httperror"
+	"github.com/rancher/rancher/pkg/settings"
+	"github.com/rancher/types/apis/management.cattle.io/v3"
+)
+
+func ValidateRancherVersion(template *v3.CatalogTemplateVersion) error {
```

Comment:
Small nitpick. Why do we need to a new folder `pkg/api/version`? Can't this function live in `pkg/catalog/utils/version`?

Reply:
I had higher aspirations for this package